README.md

```diff
@@ -1,111 +1,108 @@
+To start a new execution, you'll need to configure three parts:
+1. What LHS do you wish to scan
+2. What structure does `an` and `bn` take, and what range do you wish to scan for each coefficient
+3. How to decide if there was a match (precission wise) 
 
-### Cool examples
+you can see examples under `scripts/` folder
 
-#### e
-To find a few formulas for e, run (don't forget to be under the source/ folder when you run it)
-```python
-python main.py MITM_RF -lhs_constant e -num_of_cores 1 -lhs_search_limit 5 -poly_a_order 2 -poly_a_coefficient_max 5 -poly_b_order 2 -poly_b_coefficient_max 5
-```
+#### Left Hand Side Hash Table (`LHSHashTable`) 
+This is a data structure that holds expressions made from the required constant.
+To create a LHS object, you'll need to choose a constant, and a range for all coefficients in the expression.
+The values generated are saved to a file to reduce execution time.
 
-#### pi
-To find a few formulas for pi, run (don't forget to be under the source/ folder when you run it)
+For example, the following code will generate all Mobius transforms of `e`, with coefficients between -5 and 5.
+The generated domain is saved to `e_lhs_dept5`
 ```python
-python main.py MITM_RF -lhs_constant pi -num_of_cores 1 -lhs_search_limit 20 -poly_a_order 2 -poly_a_coefficient_max 13 -poly_b_order 3 -poly_b_coefficient_max 11 --polynomial_shift1_bn
+ from ramanujan.LHSHashTable import LHSHashTable
+ from ramanujan.constants import g_const_dict
+
+ saved_hash = 'e_lhs_dept5'
+ lhs_coefs_range = 5
+ lhs = LHSHashTable(
+    saved_hash,
+    lhs_coefs_range,
+    [g_const_dict['e']])
 ```
 
-#### Riemann Zeta function at 3 (Aépry's constant)
-To find a few formulas related to the Riemann zeta function at 3 (Zeta of 3 is called [Apéry's constant](https://www.wikiwand.com/en/Ap%C3%A9ry%27s_constant) and has a role in the electron's gyromagnetic ratio), run:
-```python
-python main.py MITM_RF -lhs_constant zeta -function_value 3 -num_of_cores 2 -lhs_search_limit 14 -poly_a_order 3 -poly_a_coefficient_max 20 -poly_b_order 3 -poly_b_coefficient_max 20 --zeta3_an --zeta_bn
-```
+#### `an` and `bn` Structures (any class under `poly_domains`) 
+An objects that generates pairs of `an` and `bn` series.
 
-#### Catalan constant
-To find a few formulas related to the [catalan constant](https://www.wikiwand.com/en/Catalan%27s_constant), you can run the following code. This one takes a bit longer generate the hash table for and make take a few minutes.
-```python
-python main.py MITM_RF -lhs_constant catalan pi-acosh_2 -num_of_cores 1 -lhs_search_limit 8 -poly_a_order 3 -poly_a_coefficient_max 15 -poly_b_order 2 -poly_b_coefficient_max 5 --catalan_bn
+The simplest structure you can choose will be `Xn = c_0 + c_1 * n + c_2 * n^2 + ...` for both `an` and `bn`, when each 
```

Comment:
it was -
n(n(...(a[0]*n + a[1]) + a[2]) + ...) + a[k]
I've update the comment so that c0 is the coef of the heighest degree and so on, I think the notation I've used is easier to understand, while the other notation is easier to compute. So I think it'd be better to keep this notation (after the updated i've added) on readme. 
What do you think?

Reply:
i agree